### PR TITLE
Minor doc fixes

### PR DESCRIPTION
**Pull request by frelon**
Signed-off-by: Fredrik Lönnegren <fredrik.lonnegren@suse.com>

### CODE REVIEW

File: docs/ami.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Amazon AMIs
 
-AMIs for RancherOS are published under the owner ID `275947076441` in the `us-west-1` and `us-west-2` regions
+AMIs for Elemental are published under the owner ID `275947076441` in the `us-west-1` and `us-west-2` regions
```
Comment (Itxaka) [inline docs/ami.md:3]: I think the Ami stuff can be fully dropped as we are not publishing anything on aws anymore and the owner I'd gas changed for sure

**Review by Itxaka — APPROVED** [review]

File: docs/ami.md (hunk 1 above)
Comment (frelon) [reply docs/ami.md:3]: Nice, will drop the whole thing!

File: docs/installation.md [hunk 2]
```diff
@@ -18,13 +18,13 @@ The installation configuration should be hosted on an HTTP or TFTP server. A sim
 ### Kernel Command Line
 
 Install directives can be set from the kernel command line using a period (.) seperated key structure such as
-`Elemental.install.config_url`.  They kernel command line keys are case-insensitive.
+`elemental.install.config_url`.  They kernel command line keys are case-insensitive.
```
Comment (fgiudici) [inline docs/installation.md:21]: Is this for cloud-config file passing via kernel cmd line 🤔 ? 
I guess this should be now:
```suggestion
`elemental.install.config-urls`.  They kernel command line keys are case-insensitive.
```
But I think @davidcassany knows better!

**Review by fgiudici — APPROVED** [review]
Ugh, we should really start taking care of the docs... thanks @frelon for improving them!

File: docs/installation.md (hunk 2 above)
Comment (frelon) [reply docs/installation.md:21]: Hmm, that might explain a few errors I've run into! I will try that on a new install and see if it works better!

Comment (frelon) [reply docs/installation.md]: Seems like this i deprecated, I removed the whole section instead :+1:

**Pull request merged**